test/AudioTest.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import chai, {expect} from 'chai';
+import {shallow, mount} from 'enzyme';
+import Audio from '../src/components/Audio';
+import Transformation from '../src/components/Transformation';
+
+chai.use(require('chai-string'));
+
+describe('Audio', () => {
+  it("should include child transformation for a single source type", function () {
+    let tag = shallow(
+      <Audio cloudName='demo'
+             sourceTypes={'webm'}
+             publicId='dog'
+             sourceTransformation={{
+               webm: {overlay: 'text:verdana_30:webm!'}
+             }}>
+        <Transformation quality='70'/>
+      </Audio>);
+    expect(tag.props().src).to.endWith('/q_70/l_text:verdana_30:webm!/dog.webm');
+  });
+
+  it("should support fps parameter", function () {
+    let tag = shallow(
+      <Audio cloudName='demo'
+             sourceTypes={'webm'}
+             publicId='dog'>
+        <Transformation fps='24'/>
+      </Audio>);
+    expect(tag.props().src).to.endWith('/fps_24/dog.webm');
+    tag = shallow(
+      <Audio cloudName='demo'
+             sourceTypes={'webm'}
+             publicId='dog'>
+        <Transformation fps='24-29.97'/>
+      </Audio>);
+    expect(tag.props().src).to.endWith('/fps_24-29.97/dog.webm');
+    tag = shallow(
+      <Audio cloudName='demo'
+             sourceTypes={'webm'}
+             publicId='dog'>
+        <Transformation fps='25-'/>
+      </Audio>);
+    expect(tag.props().src).to.endWith('/fps_25-/dog.webm');
+  });
+
+  it('should support startOffset parameter', function () {
+    let tag = shallow(
+      <Audio cloudName="demo" sourceTypes={'webm'} publicId="dog">
```

Comment:
@maoznir , please change the tests to be audio-oriented... I understand that the mechanism of setting the sourceType to webm or mp3 is identical, but still... it doesn't look good